### PR TITLE
Add axpre-suploc to iset.mm

### PR DESCRIPTION
**Pull request by jkingdon**
This is suplocsr but expressed in terms of RR rather than R.

Includes ax-pre-suploc which is a restatement using $a .

Includes lemma axpre-suploclemres

### CODE REVIEW

**Review by tirix — APPROVED** [review]

**Review by icecream17 — APPROVED** [review]

File: iset.mm [hunk 1]
```diff
@@ -81452,6 +81452,104 @@ natural numbers (notably, ~ axcaucvg ).  (Contributed by Jim Kingdon,
...
+      DYPRWAWLVRXQXRWIXCVPXSUUFUYLUYOQBUWESUXCYSUYLUUEUYOUXCYRUYKCFUXCYQUWLUXDW
+      SWBUXCUUDUYNCSUXCYTUYMUUCYOUWEYPRWAXTWBYAWLYBVS $.
+  $}
+
+  ${
+    $d A a b c d w $.  $d A a b c d x y z $.
+    $( An inhabited, bounded-above, located set of reals has a supremum.
+
+       Locatedness here means that given ` x < y ` , either there is an element
+       of ` A ` greater than ` x ` , or ` y ` is an upper bound.
```
Comment (benjub) [inline iset.mm:81522]: @jkingdon We don't know what `A` is. I'd phrase it:
```
A set of reals ` A ` is located if given any reals ... upper bound of ` A ` .
```

File: iset.mm [hunk 2]
```diff
@@ -81701,6 +81799,28 @@ A Cauchy sequence (as defined here, which has a rate convergence built
...
+  ${
+    $d A x y z $.
+    $( An inhabited, bounded-above, located set of reals has a supremum.
+
+       Locatedness here means that given ` x < y ` , either there is an element
+       of ` A ` greater than ` x ` , or ` y ` is an upper bound.
+
+       Although this and ~ ax-caucvg are both completeness properties,
+       countable choice would probably be needed to derive this from
+       ~ ax-caucvg .
```
Comment (benjub) [inline iset.mm:81811]: But this Dedekind completeness implies Cauchy completeness in IZF (even CZF), correct ? (else, I'd have to correct https://us.metamath.org/ileuni/ax-ddkcomp.html)

File: iset.mm (hunk 1 above)
Comment (jkingdon) [reply iset.mm:81522]: I think we can just change A to "the set". I'll include that in my next pull request.

Agreed that what is there now isn't very good wording.

File: iset.mm (hunk 2 above)
Comment (jkingdon) [reply iset.mm:81811]: I think so, although most of the sources don't exactly address this expressed in quite this way, so I've been a little bit weasel wording statements of this sort unless we prove something.

**Pull request merged**